src/hooks/useAdvisorProfile.ts

```
@@ -14,15 +14,21 @@ export function useAdvisorProfile() {
+      // Try the new table name first, fallback to old name during migration
+      let { data, error } = await supabase
+        .from('representatives')
         .select('*')
         .single();
-
-      // Option 2: Using the helper function
-      // const { data, error } = await supabase
-      //   .rpc('get_current_advisor_profile');
+        
+      // If representatives table doesn't exist yet, try the old table name
+      if (error && error.message.includes('relation "public.representatives" does not exist')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)